features-json/colr.json

```diff
@@ -0,0 +1,449 @@
+{
+  "title":"COLR/CPAL(v0) Font Formats",
+  "description":"The COLR table adds support for multi-colored glyphs in a manner that integrates with the rasterizers of existing text engines. COLRv0 only supports pure colors, does not support gradients, transformations and various blending modes.",
+  "spec":"https://docs.microsoft.com/en-us/typography/opentype/otspec190alpha/colr",
+  "status":"other",
+  "links":[
+    {
+      "url": "https://www.colorfonts.wtf/#w-node-85d8080e63a6-0134536f",
+      "title": "Where can I use color fonts"
+    },
+    {
+      "url": "https://docs.microsoft.com/en-us/windows/win32/directwrite/color-fonts",
+      "title": "DirectWrite's color fonts document"
+    },
+    {
+      "url":"https://pixelambacht.nl/chromacheck/",
+      "title":"A tool to check whether the browser supports OpenType color formats."
+    },
+    {
+      "url":"https://www.harbortype.com/fonts/rocher-color/",
+      "title":"A variable color font."
+    },
+    {
+      "url":"https://glyphsapp.com/learn/creating-a-microsoft-color-font",
+      "title":"Make COLR/CPAL format fonts in glyphs App."
+    }
+  ],
+  "bugs":[
+    
+  ],
+  "categories":[
+    "Other"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n"
+    },
+    "edge":{
+      "12":"y",
```

Comment:
COLR is already supported in Windows 8.1, so I believe Edge also supports it. https://www.theverge.com/2015/4/29/8511169/microsoft-edge-official-name-internet-explorer-upgrade